src/lib/lore.sh

```diff
@@ -318,13 +310,16 @@ function process_name()
 # This function resets all data structures that represent the current lore
 # fetch session. A lore fetch session is constituted by an array with the
 # latest patchsets of a lore public mailing list ordered, the number of patchsets
-# processed (the size of the array), and the oldest timeframe considered
-# (the last time period queried for patchsets).
+# processed (the size of the array), and a timeframe (from time A until time B).
+#
+# @lore_requests_timeframe: Size of the timeframe considered in days.
 function reset_current_lore_fetch_session()
 {
+  local lore_requests_timeframe="$1"
+
   list_of_mailinglist_patches=()
   PATCHSETS_PROCESSED=0
-  DAYS="$TIMEFRAME_SIZE_IN_DAYS"
+  DAYS="$lore_requests_timeframe"
```

Comment:
btw, you can still have a fallback option in case we have any parser issue or if the user set it to a weird value. I.e., set a default case if this variable is invalid.